tsconfig.module.json

```diff
@@ -2,7 +2,7 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "build": true,
+    // "build": true,
```

Comment:
This previously caused an issue, but I don't think it is anymore.
```suggestion
    "build": true,
```

Reply:
This is still causing issues: `Option '--build' must be the first command line argument`